StRoot/StFcsDbMaker/StFcsDb.h

```diff
@@ -256,6 +257,19 @@ class StFcsDb : public TDataSet {
   void setPedestal(int ehp, int ns, int dep, int ch, float ped); //! setting pedestal
   void readPedFromText(const char* file="fcsped.txt"); //! reading pedestal from text
   
+  // Tracing back from a given g2t_track_id to primary track's g2t_track_id
+  unsigned int backTraceG2tTrack(unsigned int id, g2t_track_st* g2ttrk);
+
+  // Getting pointer to parent & primary g2t_track from StFcsHit & StFcsCluster
```

Comment:
You should use doxygen-style comments for your methods -- https://doxygen.nl/manual/docblocks.html.

Reply:
Done